decentraland/unity-renderer
# Fix: Improved tutorial steps

## What does this PR change?

Fix #1949
Improves the initial and the camera switch tutorial steps

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/tutorial-improvs
2. Start the tutorial
3. Verify that the initial step works as intended
4. Verify that the camera switch from third person to first person step works both with the "V" key and the scroll wheel

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
